signing/secp256k1/impl/src/test/java/tech/pegasys/signers/secp256k1/azure/AzureKeyVaultSignerTest.java

```diff
@@ -41,7 +55,35 @@ public void azureSignerCanSignTwice() {
 
     final AzureKeyVaultSignerFactory factory = new AzureKeyVaultSignerFactory();
     final Signer signer = factory.createSigner(config);
-    signer.sign("Hello World".getBytes(UTF_8));
-    signer.sign("Hello World".getBytes(UTF_8));
+
+    final byte[] dataToHash = "Hello World".getBytes(UTF_8);
+    signer.sign(dataToHash);
+    signer.sign(dataToHash);
+  }
+
+  @Test
+  void azureWithOutHashingProducesDifferentAnswerToThatWithHashing() throws SignatureException {
```

Comment:
Can this test name be improved? Isn't this testing that the AzureKeyVaultSigner isn't hashing the data? azureWithoutHashingDoesntHashData?

Reply:
true true.